templates/base.html

```diff
@@ -11,18 +11,21 @@
 
     <meta name="theme-color" content="#191919">
 
-    <meta property="og:image" content="http{% if request.is_secure %}s{% endif %}://{{ request.get_host }}{% static 'images/black.png' %}">
+    <meta property="og:url" content="https://warsztatywww.pl{{ request.path }}" />
+    <meta property="og:locale" content="PL_pl" />
+    <meta property="og:type" content="website" />
+    <meta property="fb:app_id" content="1522345984699744" />
+    <meta property="og:image" content="http{% if request.is_secure %}s{% endif %}://{{ request.get_host }}{% static 'images/fb_image-cut.jpg' %}">
     <meta property="og:description"
-          content="Wakacyjne Warsztaty Wielodyscyplinarne to coroczna impreza organizowana przez studentów,
+          content="WWW to coroczna impreza organizowana przez studentów,
             przeznaczona dla licealistów zainteresowanych matematyką, informatyką, fizyką lub astronomią.">
-
-    <title>
-      {% if title %}
-        {{ title }} - WWW
-      {% else %}
-        Wakacyjne Warsztaty Wielodyscyplinarne
-      {% endif %}
-    </title>
+    {% if title %}
+    <title>{{ title }} - WWW</title>
+    <meta property="og:title" content="{{ title }} - WWW" />
```

Comment:
Do you want links to the program page to show up as `Program WWW16 - WWW` (without the full name, but with WWW twice?)

Reply:
Hmm, maybe let's remove `- WWW` completelly?

Reply:
In that case, workshop pages will be just the workshop title without mention of WWW. The program page definitely doesn't need it though...